src/sql/base/browser/ui/table/plugins/textWithIconColumn.ts

```diff
@@ -19,6 +19,8 @@ export interface TextWithIconColumnOptions {
 	id?: string;
 	resizable?: boolean;
 	name?: string;
+	headerCssClass?: string;
+	formatter?: Slick.Formatter<any>
```

Comment:
You should make this a generic `TextWithIconColumnOptions<T extends Slick.SlickData>` and then the formatter should be `Slick.Formatter<T>`